samples/bluetooth/mesh/light/README.rst

```diff
@@ -22,15 +22,15 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833, nrf52833dk_nrf52820, thingy53_nrf5340_cpuapp
 
-.. note::
-   When used with :ref:`zephyr:thingy53_nrf5340`, the sample supports the MCUboot bootloader with serial recovery and SMP DFU over Bluetooth.
-   Debug logs are provided over the USB CDC ACM class serial port.
-
 The sample also requires a smartphone with Nordic Semiconductor's nRF Mesh mobile app installed in one of the following versions:
 
   * `nRF Mesh mobile app for Android`_
   * `nRF Mesh mobile app for iOS`_
 
+.. note::
+   If the sample is used with Thingy:53, it complies with Thingy:53 application guide and enables additional features.
+   See :ref:`thingy53_app_guide` for details.
```

Comment:
These lines should be an entry in shortcuts.txt, to avoid duplicating them for each sample.
Also, I'm not sure how to understand "it complies with Thingy:53 application guide". How can a sample (firmware) comply with an application guide (documentation)?
```suggestion
  If you use this sample with Thingy:53, it enables additional features proper of the Thingy:53 application.
  See :ref:`thingy53_app_guide` for details.
```

Reply:
OK, now I get the meaning. So either use my propose text, which I think fits the case here, or something like `If you use ..., check the <Thingy:53 application guide> for details about the additional features that are enabled by default`. I just don't like the phrase "it complies with the guide" -- the guide is just a help, not a requirement. @wiba-nordic , what do you think?

Reply:
I agree that the phrasing you have in this comment "check the guide for details about the additional features that are enabled by default" is a clear improvement.

Reply:
I rephrased this one a bit. Would this be ok?

Reply:
Wouldnt it be more correct to state that if you build this sample for the Thingy:53 it enables additional features?

Reply:
Ok, I will update it